src/main/java/jalau/at18/kingoftokyo/DiceFaceKeeperProcess.java

```diff
@@ -0,0 +1,36 @@
+package jalau.at18.kingoftokyo;
+
+import java.util.HashMap;
+
+public class DiceFaceKeeperProcess {
```

Comment:
maybe could be better to name it DiceFaceAnalizer, because it will not be only used with the keeped dice otherwise when the player turn will finish and there should  count all the dices.